Tests/SourceKitLSPTests/SemanticTokensTests.swift

```diff
@@ -335,7 +335,7 @@ final class SemanticTokensTests: XCTestCase {
       expected: [
         TokenSpec(marker: "1️⃣", length: 4, kind: .keyword),
         TokenSpec(marker: "2️⃣", length: 1, kind: .identifier),
-        TokenSpec(marker: "3️⃣", length: 1, kind: .function),
+        TokenSpec(marker: "3️⃣", length: 1, kind: .function, modifiers: .parameterLabel),
```

Comment:
Do we also add the `parameterLabel` modifier to labels in function calls? Shouldn’t your change also affect the `SemanticTokenTests.testArgumentLabels` test?

Reply:
Yes, as it is also adding `function` token on the function call as well.

Oh, I missed that test, I'll update